resolwe/flow/static/flow/processorSchema.json

```diff
@@ -114,6 +114,32 @@
                         }
                     },
                     "additionalProperties": false
+                },
+                "resources": {
+                    "description": "Resource requirements for this process",
+                    "type": "object",
+                    "properties": {
+                        "cores": {
+                            "description": "Number of CPU cores",
+                            "type": "integer"
+                        },
+                        "memory": {
+                            "description": "Amount of memory in megabytes",
+                            "type": "integer"
+                        },
+                        "temporary_storage": {
```

Comment:
I've added them as they are also present in CWL and make sense to have at some point. But I think this is something we can add later as it is not clear how exactly we will handle them.